src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.html

```diff
@@ -157,3 +170,11 @@
      (click)="resetFilter()"
      i18n>Reset filter.</a>
 </ng-template>
+
+<ng-template #daemonLogsTpl>
+  <cd-alert-panel type="info"
+                  title="Loki/Promtail service not running"
+                  i18n-title>
+  <ng-container i18n>Please start the loki and promtail service to see these logs.</ng-container>
```

Comment:
I know there is a documentation PR open for documenting the steps to enable it. So I am assuming once its done, we will be putting the link here right? Can we somehow assume the link that documentation is going to generate and put it here?

Reply:
Even if the documentation is there, the only thing we need here is to tell the user to create the loki and promtail service, so why redirecting the user to the docs and tell the same thing?

Reply:
Okay. If it's just the service creation part then we can just give a link that redirects to the service creation form. (Same as in nfs creation form)

Reply:
Its actually just the service creation, but we need to tell the user to enable the log to files as well, which is also already documented but we can tell that thing to the user again for this, so I have added that in the updated documentation as well.

Reply:
@aaSharma14 I tested this PR, So by default **promtail** and l**oki** service won't deploy like monitoring stack services?

Reply:
@sunilangadi2 , Yes the promtail and loki service won't deploy by dfeault

Reply:
@aaSharma14 wouldn't be good if you deploy **promtail** and **loki** service by default,
users always prefer to see the each daemon logs in the dashboard, So I feel instead of manually enabling this service It's good enable it default?
what's your thought @epuertat ?